enterprise/templates/enterprise/admin/manage_learners.html

```diff
@@ -101,9 +101,9 @@
 
 <div class="forms-panel">
   <h1>{% trans "Link learner" %}</h1>
-  <form action="" method="post">
+  <form action="" method="post" enctype="multipart/form-data">
     {% csrf_token %}
-    {{ manage_learners_form }}
+    {{ manage_learners_form.as_p }}
```

Comment:
@mattdrayer "as paragraphs". Without this, it just dumped all the inputs without any structure, resulting in "whatever layout browser thinks correct". In this particular case, "Bulk CSV upload" label started on the same line as "Username or Email" input. With `as_p` it wraps each field into a paragraph (both input field and label).